js/map.js

```diff
@@ -0,0 +1,89 @@
+/* global L:readonly */
+
+import {enableElements} from './disable.js';
+import {similarListings} from './listing.js';
+import {
+  FLOAT_DIGITS,
+  TOKYO_COORDINATES,
+  ZOOM,
+  MAIN_PIN,
+  PIN
+} from './data.js';
+import {createSimilarPopup} from './new-cards.js';
+
+const addressField = document.querySelector('#address');
+
+if (window.L === undefined) {
```

Comment:
```suggestion
if (window.L) {
```
лишняя проверка, если L undefined, то if не сработает